doc/man3/EVP_DigestInit.pod

```diff
@@ -815,6 +853,8 @@ The EVP_MD_CTX_dup() function was added in OpenSSL 3.1.
 
 The EVP_DigestSqueeze() function was added in OpenSSL 3.3.
 
+The EVP_MD_CTX_get_size_ex() and EVP_xof() functions were added in OpenSSL 3.4.
```

Comment:
Seems like the redefinition of the `EVP_MD_CTX_get_size()` macro should perhaps be discussed in this history section